airbyte-integrations/connectors/source-hubspot/sample_files/incremental_catalog.json

```diff
@@ -237,23 +240,17 @@
       "cursor_field": ["updatedAt"],
       "destination_sync_mode": "append"
     },
-    {
-      "stream": {
-        "name": "ticket_pipelines",
-        "json_schema": {},
-        "supported_sync_modes": ["full_refresh"]
-      },
-      "sync_mode": "full_refresh",
-      "destination_sync_mode": "overwrite"
-    },
```

Comment:
Why not include this stream in the incremental catalog example?

Reply:
this stream is empty, while for incremental tests, all streams should produce at least 1 record